samples/LiveEffect/src/main/cpp/FullDuplexStream.cpp

```diff
@@ -112,6 +112,8 @@ oboe::Result FullDuplexStream::stop() {
     if (mInputStream) {
         inputResult = mInputStream->requestStop();
     }
+    setOutputStream(nullptr);
```

Comment:
This would prevent restarting the FullDuplexStream.

Reply:
#1143 
The inputStream and outputStream will be assigned again in LiveEffectEngine::openStreams when restarting.

Reply:
Yes, the app will reset them again when you push the button.
But some people will use this class by itself in their own apps.
If you clear the streams then calling FullDuplexStream::start() again will fail.

Reply:
I am sorry I wasn't running the latest code. This problem has been fixed in 00039f7 by removing destructor of LiveEffectEngine. It works fine now.